Marlin/temperature.cpp

```diff
@@ -1544,70 +1546,79 @@ void Temperature::isr() {
   #endif
 
   #if DISABLED(SLOW_PWM_HEATERS)
```

Comment:
Why not:
    constexpr uint8_t pwm_mask = ENABLED(SOFT_PWM_DITHER) ? _BV(SOFT_PWM_SCALE) - 1 : 0;